package.json

```diff
@@ -138,7 +138,7 @@
     "tslint": "5.14.0",
     "typescript": "^3.8.2",
     "typescript-to-proptypes": "^1.4.0",
-    "vrtest": "^0.2.0",
+    "vrtest-olivier": "^0.3.3",
```

Comment:
At this point we might as well inline it into the repository. There are a couple of blockers that would be easier by having full control over it. We could put it in `test/vrtest`, name it `vrtest` and add it to `workspaces` in the root.

Reply:
It uses https://github.com/mui-org/vrtest-olivier now. Yeah, we could potentially inline it in the repository. Long term, is this a logic we want to own or rely on a third-party?